paket.dependencies

```diff
@@ -61,10 +61,8 @@ group fakemodule
     nuget Mono.Cecil prerelease
     nuget Mono.Posix.NETStandard
     nuget FSharp.Control.Reactive
-    nuget MSBuild.StructuredLogger
-        nuget Microsoft.Build
+    nuget MSBuild.StructuredLogger ~> 2.1.768
```

Comment:
I again don't really know what I'm doing with Paket - i set this to the minium version where the dependencies changed (it no logner depends on Microsoft.Build or Microsoft.Build.Tasks.Core after this), but I'm not really sure why the explicit dependencies are here in the first place, so... :-(

Reply:
and that seems to have broken the integration tests
```
 ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Framework, Version=15.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.
File name: 'Microsoft.Build.Framework, Version=15.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
```
:-(